README.md

```diff
@@ -1,55 +1,71 @@
 # Guide
 
-A website to teach Wagtail CMS to content editors, moderators and administrators.
+Guide is a website to help content editors, moderators, administrators, and other users learn how to user the Wagtail content management system (CMS).
 
-Wagtail guide brings you:
-- Tutorials, 
-- How-to, 
-- Reference 
-- Background information
+The Wagtail guide will ultimately include:
+- Tutorials 
+- How-to articles
+- Reference materials
+- Walkthroughs and visual learning materials
 
+# Document contents
 
-## Google Summer of Code
-
-This project is a part of Google Summer of Code 2022 under the organization [**Wagtail**](https://wagtail.org/). Find more details about the project [here](https://summerofcode.withgoogle.com/programs/2022/projects/7nMw2hTq).
+- [Installation](#installation)
+- [Contributing](#contributing)
+- [Other Notes](#other-notes)
 
+# Installation
 
 ## Development
 
 We assume that you have basic knowledge of Node/Yarn/Webpack and Python/Django/Wagtail.
 
-### Frontend
+- [Virtualenv](#setup-with-virtualenv)
+- [Gitpod](#setup-with-gitpod)
```

Comment:
Might be better to present a single way to create a development environment. The most default one.

Can we present Gitpod as an evaluation tool?